src/Pages/Login.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router';
+import { Link } from 'react-router-dom';
+import { useAuth } from "../Context/AuthProvider"
+export const Login = () => {
+    const navigate = useNavigate();
+    const { loginUserWithCredentials } = useAuth()
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [showpasswordState, setPassState] = useState(false)
+    const [errorState, setErrorState] = useState(false);
+    async function loginHandler(e, email, password) {
+        e.preventDefault()
+        const response = await loginUserWithCredentials(email, password)
+        if (response.success)
+            navigate("/")
```

Comment:
navigate to the last visited page usign React Router history? It'd be easier for the user for navigation in the site